include/gridtools/fn/column_stage.hpp

```diff
@@ -101,5 +101,13 @@ namespace gridtools::fn {
     using column_stage_impl_::column_stage;
     using column_stage_impl_::fwd;
     using column_stage_impl_::merged_column_stage;
+#if defined(__NVCC__) && (__CUDACC_VER_MAJOR__ == 12 && __CUDACC_VER_MINOR__ >= 1 && __CUDACC_VER_MINOR__ <= 2)
```

Comment:
You could extract this expression and use it in al 3 places in this PR:
```suggestion
#if GRIDTOOLS_CUDA_CTAD_WORKAROUND
```

Reply:
done